Assets/Scripts/NpcBehaviors/RobotNpc.cs

```diff
@@ -80,7 +80,7 @@ protected override void EnterIdle()
     {
```

Comment:
If the EnterIdle function (or any other Enter___ function) isn't doing anything other than calling the base version of itself, it can safely be removed from the NPC script.